bindings/pyroot_experimental/cppyy/cppyy-backend/CMakeLists.txt

```diff
@@ -15,20 +15,27 @@ set(py_sources
   cppyy_backend/_get_cppflags.py
 )
 
-foreach(py_source ${py_sources})
-  file(COPY cling/python/${py_source} DESTINATION ${localruntimedir}/cppyy_backend)
-  install(FILES cling/python/${py_source} DESTINATION ${CMAKE_INSTALL_LIBDIR}/cppyy_backend)
-  install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} -m py_compile ${d}/${py_source})")
-  install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} -O -m py_compile ${d}/${py_source})")
-endforeach()
+set(cppyy_backendDirName cling/python/cppyy_backend)
+file(COPY ${cppyy_backendDirName} DESTINATION ${localruntimedir})
+
+foreach(val RANGE ${how_many_pythons})
+  list(GET python_version_strings ${val} python_version_string)
+  list(GET python_executables ${val} python_executable)
 
-add_library(cppyy_backend SHARED clingwrapper/src/clingwrapper.cxx)
-target_include_directories(cppyy_backend PRIVATE clingwrapper/src)
-target_link_libraries(cppyy_backend Core ${CMAKE_DL_LIBS})
+  foreach(py_source ${py_sources})
```

Comment:
do only once and check if format in the two versions is the same

Reply:
@etejedor Actually we can keep it as it is, compiling once for every version of Python. The reason is that Python2, when compiles, produces files with the same name at the same level of the source file with the suffix '.pyc' and '.pyo'. Python3, on the other hand, produces '.pyc' files containing the pattern 'maj.min' in the name, but inside the __pycache__ directory, meaning that the interpreter when imports knows how to pick correctly.

Reply:
Which is also the reason why I didn't observe any clash/weird behavior when I tried yesterday.

Reply:
That's great, thanks for the research @maxgalli !